control_plane/roles/control_plane_customiso/tasks/main.yml

```diff
@@ -13,7 +13,13 @@
 #  limitations under the License.
 ---
 
-# Will be updated later in each PR
-- name: Pass
-  debug:
-    msg: "Pass"
\ No newline at end of file
+# tasks file for control_plane_customiso
+
+- name: Check iso mount path
+  include_tasks: check_prerequisites.yml
+
+- name: Edit iso config files
+  include_tasks: edit_iso_config.yml
+
+- name: Create unattended iso file
+  include_tasks: create_unattended_iso.yml
```

Comment:
@sujit-jadhav and @abhishek-s-a, do we intend to use the same unattended ISO for both iDRAC and PXE?

Reply:
unattended ISO file is used only for iDRAC. For PXE boot using cobbler profile with mounted ISO files.